index.html

```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>project #1</title>
+</head>
+<body>
+  <h1>Hello world!</h1>
+</body>
+</html>
```

Comment:
```suggestion
</body>
</html>

```
в конце файла должна быть пустая строка